agent/engine/common_integ_test.go

```diff
@@ -117,6 +117,17 @@ func verifyContainerRunningStateChangeWithRuntimeID(t *testing.T, taskEngine Tas
 		"Expected container runtimeID should not empty")
 }
 
+func verifyExecAgentRunningStateChange(t *testing.T, taskEngine TaskEngine) {
+	stateChangeEvents := taskEngine.StateChangeEvents()
+	event := <-stateChangeEvents
+	containerEvent := event.(api.ContainerStateChange)
+	assert.NotEmpty(t, containerEvent.ManagedAgents, "Expeceted exec-agent event has no managed agents")
+	if containerEvent.ManagedAgents != nil {
+		assert.Equal(t, apicontainerstatus.ManagedAgentRunning, containerEvent.ManagedAgents[0].Status,
+			"expected managedAgent container state change event did not match actual event")
```

Comment:
nit: this error msg is pretty much what `assert.Equal()` means, maybe we don't need it here =)

Reply:
We need it for debugging a failed test.  It may also be useful if we extend this function to check for other types of state change events.